src/app/component/Typography.jsx

```diff
@@ -1,40 +1,60 @@
-import { Typography } from "@material-ui/core";
-import React from "react";
+//import { MuiTypography, variantMapping } from "@material-ui/core";
+//import React from "react";
+import { createMuiTheme } from "@material-ui/core/styles";
 
-const H1 = ({ children, ...rest }) => (
-  <Typography variant="h1" {...rest}>
-    {children}{" "}
-  </Typography>
-);
-const H2 = ({ children, ...rest }) => (
-  <Typography variant="h2" {...rest}>
-    {children}
-  </Typography>
-);
-const H3 = ({ children, ...rest }) => (
-  <Typography variant="h3" {...rest}>
-    {children}{" "}
-  </Typography>
-);
-const H4 = ({ children, ...rest }) => (
-  <Typography variant="h4" {...rest}>
-    {children}{" "}
-  </Typography>
-);
-const H5 = ({ children, ...rest }) => (
-  <Typography variant="h5" {...rest}>
-    {children}{" "}
-  </Typography>
-);
-const Body1 = ({ children, ...rest }) => (
-  <Typography variant="body1" {...rest}>
-    {children}{" "}
-  </Typography>
-);
-const Body2 = ({ children, ...rest }) => (
-  <Typography variant="body2" {...rest}>
-    {children}{" "}
-  </Typography>
-);
+// const H1 = ({ children, ...rest }) => (
+//   <Typography variant="h1" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
+// const H2 = ({ children, ...rest }) => (
+//   <Typography variant="h2" {...rest}>
+//     {children}
+//   </Typography>
+// );
+// const H3 = ({ children, ...rest }) => (
+//   <Typography variant="h3" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
+// const H4 = ({ children, ...rest }) => (
+//   <Typography variant="h4" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
+// const H5 = ({ children, ...rest }) => (
+//   <Typography variant="h5" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
+// const Body1 = ({ children, ...rest }) => (
+//   <Typography variant="body1" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
+// const Body2 = ({ children, ...rest }) => (
+//   <Typography variant="body2" {...rest}>
+//     {children}{" "}
+//   </Typography>
+// );
 
-export { H1, H2, H3, H4, H5, Body1, Body2 };
+// export { H1, H2, H3, H4, H5, Body1, Body2 };
+
+export const theme = createMuiTheme({
```

Comment:
Just looking at the changes you're suggesting here, by the looks of the file, we're expecting to export Typography components. The change you've made is exporting a theme object.

This definitely will break implementation of this file. I'm 95% sure the variantMapping object you're adding in to the createMuiTheme method should be in the app level theme object that we're probably setting up somewhere else in the application.

Reply:
I realize that my implementation was incorrect as well. I attempted to try and keep it somewhat similar to the original only to realize I can't return an object inside of a const method as I expected. I think what I will do is close the PR and try to rethink the strategy. What do you think?